openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -61,27 +61,33 @@ def create_jwt_from_token(token_dict, oauth_adapter, use_asymmetric_key=None):
     access_token = oauth_adapter.get_access_token(token_dict['access_token'])
     client = oauth_adapter.get_client_for_token(access_token)
 
-    # .. setting_name: JWT_ACCESS_TOKEN_EXPIRE_SECONDS
-    # .. setting_default: 60 * 60
-    # .. setting_description: The number of seconds a JWT access token remains valid. We use this
-    #     custom setting for JWT formatted access tokens, rather than the django-oauth-toolkit setting
-    #     ACCESS_TOKEN_EXPIRE_SECONDS, because the JWT is non-revocable and we want it to be shorter
-    #     lived than the legacy Bearer (opaque) access tokens, and thus to have a smaller default.
-    # .. setting_warning: For security purposes, 1 hour (the default) is the maximum recommended setting
-    #     value. For tighter security, you can use a shorter amount of time.
-    token_dict['expires_in'] = getattr(settings, 'JWT_ACCESS_TOKEN_EXPIRE_SECONDS', 60 * 60)
```

Comment:
**Note to Reviewer:** When I added this code earlier, I should not have made this destructive change to the `token_dict`. I had misread the original code and thought this was updating all parts of the `token_dict`, and I now see where that is actually updated (for now).